src/swig_python/contrib/gdk_example_liquid_amp.py

```diff
@@ -35,7 +35,7 @@ def main():
     """
     # To login to an existing wallet you can either use the mnemonic or pin.
     # Later we'll see how to use a pin, for now we will use the mnemonic.
-    mnemonic = 'your twenty four word mnemonic goes here with single spaced words'
+    mnemonic = 'your twenty four of twelve word mnemonic goes here with single spaced words'
```

Comment:
I think you mean `or twelve`, but `Your twelve or twenty four word...` reads better.